pages/expense.js

```diff
@@ -79,6 +79,10 @@ const expensePageQuery = gqlV2/* GraphQL */ `
           ...CommentFields
         }
       }
+      account {
+        id
+        slug
+      }
```

Comment:
The `ExpensePageExpenseFields` fragment already fetches that. Do we really need that?